nasa-gibs/worldview
# WV-2851: Add 9 GEDI layers

## Description

Fixes #WV-2851.

Added 9 new GEDI layers

1. GEDI_ISS_L4B_Aboveground_Biomass_Density_Mean_201904-202303
2. GEDI_ISS_L4B_Aboveground_Biomass_Density_StandardError_201904-202303
3. GEDI_ISS_L4B_Aboveground_Biomass_Density_PercentStandardError_201904-202303
4. GEDI_ISS_L4B_Number_of_Samples_201904-202303
5. GEDI_ISS_L3_Elevation_Mean_Lowest_Mode_201904-202303
6. GEDI_ISS_L3_Elevation_StdDev_Lowest_Mode_201904-202303
7. GEDI_ISS_L3_Laser_Footprint_Count_201904-202303
8. GEDI_ISS_L3_Canopy_Height_Mean_RH100_201904-202303
9. GEDI_ISS_L3_Canopy_Height_StdDev_RH100_201904-202303

## How To Test

1. Open with [these URL parameters](http://localhost:3000/?v=-110.5305522900622,-44.587578091011224,79.98299925022216,54.974921908988776&l=GEDI_ISS_L3_Canopy_Height_StdDev_RH100_201904-202303,GEDI_ISS_L3_Canopy_Height_Mean_RH100_201904-202303,GEDI_ISS_L3_Elevation_StdDev_Lowest_Mode_201904-202303,GEDI_ISS_L3_Elevation_Mean_Lowest_Mode_201904-202303,GEDI_ISS_L3_Laser_Footprint_Count_201904-202303,GEDI_ISS_L4B_Number_of_Samples_201904-202303,GEDI_ISS_L4B_Aboveground_Biomass_Density_PercentStandardError_201904-202303,GEDI_ISS_L4B_Aboveground_Biomass_Density_StandardError_201904-202303,GEDI_ISS_L4B_Aboveground_Biomass_Density_Mean_201904-202303,MODIS_Terra_CorrectedReflectance_TrueColor&lg=true&t=2021-02-08-T14%3A44%3A04Z)
2. Check that each new GEDI layer displays on the map and that you are able to take an image snapshot of them.



## PR Submission Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/main/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
